ajax/libs/Zebra_Datepicker/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Zebra_Datepicker",
+  "filename": "zebra_datepicker.js",
+  "version": "1.9.5",
+  "homepage": "http://www.stefangabos.ro/jquery/zebra-datepicker/",
+  "author": {
+    "name": "Stefan Gabos",
+    "email": "contact@stefangabos.ro"
+  },
+  "description": "A super-lightweight, highly configurable, cross-browser date picker jQuery plugin",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stefangabos/Zebra_Datepicker.git"
+  },
+  "keywords": [
+    "calendar",
+    "date",
+    "datepicker",
+    "form",
+    "iso-8601",
+    "ui"
+  ],
+  "license": "LGPL-3.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/stefangabos/Zebra_Datepicker.git",
+    "fileMap": [
+      {
+        "basePath": "public/javascript",
```

Comment:
Wrong `basePath`. Could you use auto-update to check what files what you grab? I think this `basePath` can't get the files under `css/` and `images/`.